packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -258,6 +260,36 @@ const components = [
       />,
     ],
   },
+  {
+    id: 'withBannerAlertState',
+    title: 'withBannerAlertState',
+    blurb: [
+      <Paragraph>
+        Most common use of &quot;expandable&quot; and &quot;dismissable&quot; can be easily achieved
+        using the &quot;withBannerAlertState&quot; HOC.
+      </Paragraph>,
+      <Paragraph>
+        It also adds the option to automatically dismiss a banner after a certain period of time has elapsed.
+      </Paragraph>,
+    ],
+    examples: [
```

Comment:
I'm not sure there's much point in showing these examples - I think it's probably enough to just show how to use the HOC in the readme.
But it's a minor point and doesn't really matter. The HOC works really well 👍

Reply:
Up to you, it's your documentation 😂 

If you want, let's see what other Backpack members think, and I'll update it accordingly, no problem :-)

I've just pushed a rebased version with latest master changes (conflicts in `changelog.md`)

Reply:
The consensus seems to be to remove the new examples, as we already have examples of the functionality the HOC provides.
But the documentation of the HOC usage is good and should stay!

Reply:
Done! I've just removed it :-)

Anything else, just let me know :-)

Reply:
All LGTM 👍 
Happy to merge!